server/trillian_log_signer/main.go

```diff
@@ -48,6 +48,8 @@ import (
 var (
 	mySQLURI                 = flag.String("mysql_uri", "test:zaphod@tcp(127.0.0.1:3306)/test", "Connection URI for MySQL database")
 	httpEndpoint             = flag.String("http_endpoint", "localhost:8091", "Endpoint for HTTP (host:port, empty means disabled)")
+	tlsCertFile              = flag.String("tls_cert_file", "", "Path to the TLS server certificate.")
```

Comment:
Perhaps add some text to say not configuring/setting and tlsKeyFile to empty results in a non-TLS server being run?

Reply:
Done.